app/views/layouts/_form_errors.html.erb

```diff
@@ -5,7 +5,7 @@
       <div class="govuk-error-summary__body">
         <ul class="govuk-list govuk-error-summary__list">
           <% resource.errors.full_messages.each do |message| %>
-            <li><%= message %></li>
+            <li><%= message.html_safe %></li>
```

Comment:
Please don't do this. it will lead to potential for a XSS attack, as this is being set broadly.
Instead, apply `html_safe` on the string with html in itself (`./app/controllers/users/registrations_controller.rb:30`)

Reply:
Notably, https://github.com/alphagov/govwifi-admin/pull/513 is committing this sin as well - we should block both if we're going to have this discussion.